packages/graphback-runtime/src/createRuntimeServices.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLObjectType, GraphQLSchema } from "graphql"
+import { PubSubEngine } from 'graphql-subscriptions';
+import { GraphbackDataProvider } from './data/GraphbackDataProvider';
+import { CRUDService } from './service/CRUDService';
+import { GraphbackModelConfig } from "./GraphbackModelConfig";
+
+export type RuntimeServicesOptions<Provider extends GraphbackDataProvider, Service extends CRUDService> = {
+  schema: GraphQLSchema,
+  models: GraphbackModelConfig[]
```

Comment:
This should be gone as it was only codegen artifacts and it makes no sense for runtime. Runtime reads our annotations and markers and have access to the global crud options